src/java.base/share/classes/javax/crypto/Mac.java

```diff
@@ -144,8 +144,8 @@ public final String getAlgorithm() {
      *
      * <p> This method traverses the list of registered security Providers,
      * starting with the most preferred Provider.
```

Comment:
Change "Providers"/"Provider" to start with lowercase.

Reply:
Fixed.